### PR TITLE
test(rust/esbuild/dce): enable json_loader_remove_unused case.

### PR DESCRIPTION
**Pull request by Cecil0o0**

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
Please paste related esbuild snapshots, or It is hard to verify.

File: crates/rolldown/tests/esbuild/dce/.tree_shaking_class_property/_config.json
Comment (IWANABETHATGUY) [inline crates/rolldown/tests/esbuild/dce/.tree_shaking_class_property/_config.json:1]: why do you delete this test case?

**Comment by Cecil0o0** [comment]
> Please paste related esbuild snapshots, or It is hard to verify.

================================================================================
TestJSONLoaderRemoveUnused
---------- /out.js ----------
// entry.js
console.log("unused import");

================================================================================

File: crates/rolldown/tests/esbuild/dce/.tree_shaking_class_property/_config.json
Comment (Cecil0o0) [reply crates/rolldown/tests/esbuild/dce/.tree_shaking_class_property/_config.json:1]: a mistake, I will revert the changes tomorrow morning

**Review by IWANABETHATGUY — APPROVED** [review]

**Pull request merged**